src/NexusMods.App.UI/Controls/ModInfo/ViewModFiles/ViewModFilesView.axaml

```diff
@@ -0,0 +1,70 @@
+﻿<reactiveUi:ReactiveUserControl
+    d:DesignHeight="450"
+    d:DesignWidth="800"
+    mc:Ignorable="d"
+    x:Class="NexusMods.App.UI.Controls.ModInfo.ViewModFiles.ViewModFilesView"
+    x:TypeArguments="viewModFiles:IViewModFilesViewModel"
+    xmlns="https://github.com/avaloniaui"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:reactiveUi="http://reactiveui.net"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:viewModFiles="clr-namespace:NexusMods.App.UI.Controls.ModInfo.ViewModFiles"
+    xmlns:resources="clr-namespace:NexusMods.App.UI.Resources">
+    <Design.DataContext>
+        <!-- This ViewModel has a bool switch 'showMultipleRoots' to toggle a view which also has Saves -->
+        <viewModFiles:ViewModFilesDesignViewModel />
+    </Design.DataContext>
+    <!-- In this view specifically, we want to shift the text of the TreeDataGrid 
+         such that we align the 'name' field with the
+    -->
+    <UserControl.Styles>
+        <!-- Note: Can't use nth-child on textblock itself. Not even actual control source does that. -->
+        <Style Selector="TreeDataGridColumnHeader:nth-child(1)">
+            <Style Selector="^ TextBlock">
+                <Setter Property="Margin" Value="24, 0,0, 0" />
+            </Style>
+        </Style>
+    </UserControl.Styles>
```

Comment:
Are you sure we might need that?
Here it's a bit of a special case because we want to align the `LOCATION` text with the `NAME` text in the first column's header. I'm not sure if that'll be a common occurrence in other views in the future.

Reply:
Lau said he liked headers to go end to end, so that means we need the padding in the first header in all TreeDataGrids that go end to end, so having a class for that would be nice.

Also I'm general you should define styles like that in the Theme.